src/main.rs

```diff
@@ -18,5 +19,18 @@ fn trace_panics() {
     std::panic::set_hook(Box::new(move |panic_info| {
         tracing_panic::panic_hook(panic_info);
         prev_hook(panic_info);
+
+        // we are already in a panic hook, if that panics as well... then we are
+        // entering a recursion loop so we're just ignoring the result
+        let _ = Notification::new()
+            .appname(clap::crate_name!())
+            .urgency(notify_rust::Urgency::Critical)
+            .summary("Flakeshot paniced!")
+            .body(concat![
+                "You likely found a bug. ",
+                "Please take a look into the log file (see `flakeshot -h`) and ",
+                "create an issue if suitable."
```

Comment:
Don't you want to add the panic message? Also we shouldn't suggest that any panic is a bug. What if the drive is full or something?

Reply:
> Don't you want to add the panic message?

No, because because the user might not be able to read everything as a notification (depending on how big the panic message will be), hence the hint to open up the logs.

Reply:
> Also we shouldn't suggest that any panic is a bug. What if the drive is full or something?

That's why I added the part

> (...) and create an issue if suitable.